### DISCUSSION TITLE
WinSSPI auth dsn example

### DISCUSSION THREAD

**Discussion by mihaibudurescu** [burningalchemist/sql_exporter#652] [Show and tell]
Hello,
I would like to add this text block in the readme file, it's an example can really help other people.
I tried via feature branch but I don't have write permissions.
To be added somewhere in  Data Source Names section, what do you think?

"Another aproach on windows is to use WinSSPI, having sql_exporter installed as a service with configured domain account and password, so without the need to expose the password in clear text in exporter configuration:
`sqlserver://@<HOST>:<PORT>?authenticator=winsspi`"

**Comment by burningalchemist** [comment]
Hey @mihaibudurescu,

Yeah, absolutely! If you don't want to spend much time on forking the project, and creating a pull request, I can open a pull request myself, and ask for your feedback on changes. Would this work for you? 👍

**Reply by mihaibudurescu** [reply]
yes, great. thank you!

**Reply by burningalchemist** [reply]
Here we go: https://github.com/burningalchemist/sql_exporter/pull/653
